flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/container/FlinkContainersBuilder.java

```diff
@@ -163,6 +163,7 @@ public FlinkContainers build() {
         this.conf.set(
                 CheckpointingOptions.CHECKPOINTS_DIRECTORY,
                 CHECKPOINT_PATH.toAbsolutePath().toUri().toString());
+        this.conf.set(RestOptions.BIND_ADDRESS, "0.0.0.0");
```

Comment:
isn't this the default (from a ConfigOption perspective)?

Reply:
Yes. Would it be better to just unset the parameter rather than hardcode it to `0.0.0.0`?

Reply:
@zentol this actually an override for `flink-conf.yaml`, see `FlinkImageBuilder#createTemporaryFlinkConfFile` for more details. So unless this is explicitly set, the override won't work.

Reply:
I mostly did it this way to mimic what we do on flink-docker, which was set it explicitly to `0.0.0.0`.

Reply:
aaaah this is for the testcontainer tests...yeah then I suppose we need it.
...
although they should rely on our docker images 💢

Reply:
At present, I try to set rest.bind-address to 0.0.0.0 on Flink 1.15 and above, and find that external access is not effective. However, the settings are valid in versions below 1.15